include/zephyr/drivers/dac/dac_esp32.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DRIVERS_DAC_ESP32_H_
+#define ZEPHYR_INCLUDE_DRIVERS_DAC_EPS32_H_
```

Comment:
There is a typo in here, please check.

Reply:
Anyway, after removing this file which is CW related, this won't be part of this PR as I see.

Reply:
file was removed